### PR TITLE
REVSDL-2374 VR plugin frame

### PR DESCRIPTION
**Pull request by VladSemenyuk**

### CODE REVIEW

**Comment by bonewell** [comment]
Will we use master branch for GPB solution?

File: src/components/vr_module/include/vr_module/vr_module.h [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    return next_correlation_id_++;
+  }
+ private:
+
+  /**
+   * @brief Subscribes plugin to mobie rpc messages
+   */
+  void SubscribeToRpcMessages();
+
+ private:
```
Comment (bonewell) [inline src/components/vr_module/include/vr_module/vr_module.h]: unnecessary

File: src/components/vr_module/include/vr_module/vr_module.h [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#ifndef SRC_COMPONENTS_VR_MODULE_INCLUDE_VR_MODULE_IVDCM_H_
```
Comment (bonewell) [inline src/components/vr_module/include/vr_module/vr_module.h]: Wrong guard define. IVDCM_H_ -> VR_MODULE_H_

File: src/components/vr_module/src/vr_module.cc [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+  // TODO(VSemenyuk): Subscribe to rpc messages here
+}
+
+PluginInfo VRModule::GetPluginInfo() const {
+  LOG4CXX_AUTO_TRACE(logger_);
+  return plugin_info_;
+}
+
+ProcessResult VRModule::ProcessMessage(application_manager::MessagePtr msg) {
+  DCHECK(msg);
```
Comment (bonewell) [inline src/components/vr_module/src/vr_module.cc]: It will be good to add LOG4CXX_AUTO_TRACE(logger_);

**Comment by bonewell** [comment]
Reviewed.

File: src/components/vr_module/src/vr_module.cc (hunk 3 above)
Comment (VladSemenyuk) [inline src/components/vr_module/src/vr_module.cc]: Will be added

File: src/components/vr_module/include/vr_module/vr_module.h (hunk 2 above)
Comment (VladSemenyuk) [inline src/components/vr_module/include/vr_module/vr_module.h]: Will be corrected

**Pull request merged**